feat: add "Enforcing these Standards" section (#92)
mediamonks/frontend-coding-standards: README.md

9. [Recommended Libraries](#recommended-libraries)
10. [Definition of Done Checklist](#definition-of-done-checklist)

## Enforcing these Standards

> In order to enforce these standards in your code, we **highly recommend** using

Reviewer: I'd say it's **required** to use eslint and prettier and not _recommended_ 😉 

```suggestion
In order to enforce these standards in your code, it is required to use
```
